lib/src/test/kotlin/starknet/data/TypedDataTest.kt

```diff
@@ -551,6 +551,14 @@ internal class TypedDataTest {
                 "revision": 1
             }
         """.trimIndent()
+        val domainObjectV1WithStringRevision = """
```

Comment:
I've decided to place domain object with revision here. It could be also placed directly in the test, but wanted to maintain consistency with the place where `domainObjectV1` is defined.

Reply:
I think we should move it directly to the test